ground/src/test/java/com/google/android/ground/ui/datacollection/tasks/multiplechoice/MultipleChoiceItemTest.kt

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.android.ground.ui.datacollection.tasks.multiplechoice
+
+import com.google.android.ground.BaseHiltTest
+import com.google.android.ground.model.task.MultipleChoice.Cardinality
+import com.google.android.ground.model.task.Option
+import com.google.common.truth.Truth.assertThat
+import dagger.hilt.android.testing.HiltAndroidTest
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+
+
+@HiltAndroidTest
+@RunWith(RobolectricTestRunner::class)
+class MultipleChoiceItemTest: BaseHiltTest() {
+
+  private val multipleChoiceItemOne = MultipleChoiceItem(
+    option = Option(
+      id = "id",
+      code = "code",
+      label = "label"
+    ),
+    cardinality = Cardinality.SELECT_ONE
+  )
+
+  private val multipleChoiceItemMultiple = MultipleChoiceItem(
+    option = Option(
+      id = "id",
+      code = "code",
+      label = "label"
+    ),
+    cardinality = Cardinality.SELECT_MULTIPLE
+  )
+
+  private val multipleChoiceItemOneId = MultipleChoiceItem(
+    option = Option(
+      id = "new id",
+      code = "code",
+      label = "label"
+    ),
+    isSelected = true,
+    cardinality = Cardinality.SELECT_MULTIPLE
+  )
+
+
+  @Test
+  fun testIsTheSameItemTrue() {
```

Comment:
Use string function name for tests, e.g. 
```
fun `isTheSameItem returns true when IDs are the same`()
```

See: https://github.com/google/ground-android/blob/a410aff6e2ea2f5f44a3a52418515507d9b44e61/ground/src/test/java/com/google/android/ground/model/task/ConditionTest.kt#L56

Also below.

Reply:
Done